### PR TITLE
Adding support for DATA attachments

### PR DESCRIPTION
**Pull request by icappello**
Adds support to send local files as attachments

### CODE REVIEW

File: lib/bamboo/adapters/sendin_blue_adapter.ex [hunk 1]
```diff
@@ -121,30 +124,34 @@ defmodule Bamboo.SendinBlueAdapter do
...
     body
     |> Map.put(:from, [address, name])
   end
 
   defp put_to(body, %Email{to: to}) do
     {names, addresses} = Enum.unzip(to)
-    body
-    |> put_addresses(:to, %{Enum.join(addresses, ",") => Enum.join(names, ",")})
+    result = body |> put_addresses(:to, %{Enum.join(addresses, ",") => Enum.join(names, ",")})
+    result
```
Comment (gpad) [inline lib/bamboo/adapters/sendin_blue_adapter.ex]:
```suggestion
    body |> put_addresses(:to, %{Enum.join(addresses, ",") => Enum.join(names, ",")})
```

File: test/lib/adapters/sendin_blue_adapter_test.exs [hunk 2]
```diff
@@ -123,8 +128,37 @@ defmodule Bamboo.SendinBlueAdapterTest do
...
+             File.read!("./test/support/attachment.png") |> Base.encode64()
+
+    assert params |> get_in(["attachment", "attachment2.png"]) ==
+             File.read!("./test/support/attachment.png") |> Base.encode64()
+  end
+
   defp new_email(attrs \\ []) do
     attrs = Keyword.merge([from: "foo@bar.com", to: []], attrs)
-    Email.new_email(attrs) |> Bamboo.Mailer.normalize_addresses
+    attrs |> Email.new_email() |> Bamboo.Mailer.normalize_addresses()
```
Comment (gpad) [inline test/lib/adapters/sendin_blue_adapter_test.exs]:
```suggestion
    Keyword.merge([from: "foo@bar.com", to: []], attrs) |> Email.new_email() |> Bamboo.Mailer.normalize_addresses()
```

Comment (icappello) [reply test/lib/adapters/sendin_blue_adapter_test.exs]: It was like this in the original library, I'll change it

File: lib/bamboo/adapters/sendin_blue_adapter.ex (hunk 1 above)
Comment (icappello) [reply lib/bamboo/adapters/sendin_blue_adapter.ex]: Ok

**Pull request merged**